ohara-manager/client/src/components/common/Mui/Typography/Typography.js

```diff
@@ -18,9 +18,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Typography from '@material-ui/core/Typography';
 
-const MuiTypography = ({ variant, children, ...rest }) => {
+const MuiTypography = ({
+  variant,
+  children,
+  testId = 'typo-graphy',
```

Comment:
This is actually a real English word :)
```suggestion
  testId = 'typography',
```

Reply:
orz, got it, lol